scripts/validate-notebooks.ps1

```diff
@@ -58,16 +58,33 @@ function Validate {
     # Convert %kata to %check_kata
     (Get-Content $Notebook -Raw) | ForEach-Object { $_.replace('%kata', '%check_kata') } | Set-Content $CheckNotebook -NoNewline
 
+    # For build 0.12.20082513, work around an IQ# package load timing issue by forcing all necessary packages to be loaded up-front.
```

Comment:
Do we need to do this check separately for each kata, or can we do it once before we start iterating over individual notebooks?